src/components/data/DataTable.js

```diff
@@ -497,6 +501,17 @@ const DataTable = props => {
       },
       onDismiss: () => setUpdatingEntity(undefined)
     }),
+    !!renamingColumn && h(RenameColumnModal, {
+      namespace, name,
+      entityType: _.head(entities).entityType,
```

Comment:
```suggestion
      entityType,
```

I notice that this is consistent with other code, but alternatively, the `entityType` prop passed to DataTable can be used here vs getting it from the entities list.